src/hidimstat/conditional_sampling.py

```diff
@@ -0,0 +1,180 @@
+from typing import Union
```

Comment:
This class will be only used by CPI. In consequence, it should be better to move it to the conditional_permutation_importance.py file.
Or to rename the file with an under bar in front of the name.

Reply:
After a short exchange with @AngelReyero, the best is to have a separate file. 
Don't change the name for the moment. This will be addresses latter with the creation of a subpackages for statistical tools.

Reply:
see issue #134